libs/utils/executor.py

```diff
@@ -321,13 +322,34 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
```

Comment:
We updated the way logging is used since #237, you should use this syntax now:

```python
self._log.debug('Configuration not provided')
```
Notice we also removed the odd %14s namespece field... ;-)